.eslintrc.js

```diff
@@ -67,6 +67,7 @@ module.exports = {
     'react/react-in-jsx-scope': 2, // Prevent missing React when using JSX
     'react/self-closing-comp': 2, // Prevent extra closing tags for components without children,
     'react/prefer-stateless-function': 2, // Use functional components vs classes
+    'react-native/no-raw-text': [2, { skip: 'Typography' }],
```

Comment:
This could be a problem, it assumes `<Typography>` translates things, which it doesn't.

Prefer the inline overrides so that the developer can put a reason why such a string it is not translated.

In the case of feature flags toggle screen, we're ok not putting this burden on translators for something not seen outside in the real world.